core/templates/dev/head/services/explorationContextService.js

```diff
@@ -100,6 +100,21 @@ oppia.factory('explorationContextService', [
       isInExplorationEditorMode: function() {
         return (this.getPageContext() === PAGE_CONTEXT.EDITOR &&
             this.getEditorTabContext() === EDITOR_TAB_CONTEXT.EDITOR);
+      },
+
+      // Following functions help in remembering whether video is already
+      // autoplayed or not. Consider the case when same video is to be shown
+      // in exploration twice, then 2nd time video will not autoplay. To
+      // solve that problem, unique id to each element can be assigned and
+      // that id can be used to store whether video is autoplayed or not.
+      addToPlayedList: function(
+          videoId) {
```

Comment:
This can be moved back up to the previous line.

Also, maybe call the method addAutoplayedVideo(), and the following one hasVideoBeenAutoplayed()